editor/src/core/model/element-metadata-utils.ts

```diff
@@ -1134,12 +1134,10 @@ export const MetadataUtils = {
         let componentInstance = spyElem.componentInstance || domElem.componentInstance
         let jsxElement = alternativeEither(spyElem.element, domElem.element)
 
-        const possibleUID: string | null | undefined = Utils.defaultIfNull(
-          spyElem.props[UTOPIA_UID_KEY],
-          spyElem.props[UTOPIA_ORIGINAL_ID_KEY],
-        )
-        if (possibleUID != null) {
-          const possibleElement = elementsByUID[possibleUID]
+        const possibleUIDs: Array<string> | null = uidsFromString(spyElem.props[UTOPIA_UIDS_KEY]) // TODO what about original-uid???
```

Comment:
I think `original-uid` will also have to become a space separated list, but that would mean all elements would need it (since right now we only add it for generated elements, meaning the length of the uid arrays would become out of sync). But then that brings about the question of whether we still need them, since we're already using a function for converting a dynamic uid to a static one. I _think_ we don't.

Reply:
I also think we don't, so long as we stick to the `~~~index` scheme of indicating a generated uid